Kickstarter-iOS/Views/Cells/DiscoveryPostcardCell.swift

```diff
@@ -127,6 +159,13 @@ internal final class DiscoveryPostcardCell: UITableViewCell, ValueCell {
       |> UIStackView.lens.layoutMargins
         .~ .init(top: Styles.grid(2), left: Styles.grid(2), bottom: 0.0, right: Styles.grid(2))
       |> UIStackView.lens.layoutMarginsRelativeArrangement .~ true
+
+    _ = self.starButton
+      |> saveButtonStyle
+      |> UIButton.lens.hidden .~ true
```

Comment:
assuming this `hidden` value is just here because the star hasn't been implemented yet, but just leaving a note to make sure it doesn't end up in `bindStyles` later!